proposals_versus_issues.md

```diff
@@ -0,0 +1,37 @@
+The purpose of this document is to illustrate the differences between an SDL Evolution Proposal and an SDL Issue.  While determining how to present an item to the SDLC can be subjective, it’s our hope that providing examples of SDL Evolution Proposals and SDL Issues will help you to identify which format and GitHub Repository should be used for bringing your issue to the attention of the SDLC Steering Committee.
+
+###SDL Evolution Proposals: [https://github.com/smartdevicelink/sdl_evolution](https://github.com/smartdevicelink/sdl_evolution)
+
+####Qualifications:
+- If there are multiple ways to approach an issue” to “If the behavior of the system has changed in a way that impacts what the app developer sees.
```

Comment:
Revised to: If the behavior of the system has changed in a way that impacts what the app developer sees.